Firmware_Bootstrap_Loader/README.md

```diff
@@ -0,0 +1,78 @@
+# Progaming Faraday - Bootstrap Loader (Windows)
+
+Faraday can be re-programmed using 2 methods:
+
+* JTAG connection
+  * Programming + debugging
+* Bootstrap Loader (USB)
+  * Programming only
+
+This tutorial will focus on programming your Faraday using only the USB connection to a computer. Using the Bootstrap Loader (BSL) is a convienient way to upgrade firmware and perform basic firmware development when debugging features are not required.
+
+> Note: Ensure Proxy is NOT running when trying to program using BSL
+
+This tutorial and version of Faraday BSL has only been tested on Windows. Future efforts will bring this capability to other operating systems.
+
+
+###Prerequisits
+ * Faraday radio connected via USB to the computer
+ * Proxy configured and running
+ * Firmware HEX file in TI-TXT format
+
+## Getting Firmware HEX Files
+
+BSL requires the use of a HEX file containing the intended firmware that will be programmed to the device in TI-TXT format. This tutorial assumes two methods:
+
+* Obtained a firmware file from a pre-made source such as Faraday's GitHub repository
```

Comment:
Honestly just link to the file or folder or tag in the firmware repository. We should settle on some tag or latest master name that's always used.